flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1115,33 +1217,8 @@ under the License.
 			<properties>
 				<hive.version>3.1.3</hive.version>
 				<derby.version>10.14.1.0</derby.version>
-				<!-- need a hadoop version that fixes HADOOP-14683 -->
-				<hivemetastore.hadoop.version>2.8.2</hivemetastore.hadoop.version>
```

Comment:
@MartijnVisser Hi, I found in here, we remove `hivemetastore.hadoop.version`, 
But before the modification, for hive3 profile, it'll still use hadoop-common 2.8.2 which is same to  `hivemetastore.hadoop.version`. That's how it currently works with Hadoop 3.
 I think we can have two solutions:
1: revert such change about this part
2: upgrad the dependency `com.klarna:hiverunner` to 6.0.1 in profile hive3, I have tried in my local and run this test `HiveRunnerITCase#testOrcSchemaEvol`. It turns out it works.
But I haven't run the whole tests.

Reply:
Oh, I miss something. If upgrade the dependency com.klarna:hiverunner to 6.0.1 in profile hive3, the compile will fail for the interfaces changed in 6.0.1.

Reply:
Ah I missed your comment but found that too indeed. I've reintroduced the parameter. I'm a bit surprised that Hive3 requires Hadoop2 and doesn't work with Hadoop 3 to be honest.

Reply:
> If upgrade the dependency com.klarna:hiverunner to 6.0.1 in profile hive3, the compile will fail for the interfaces changed in 6.0.1.

Yes, Hiverunner has changed their Maven groupId. Version 6 is compatible with Hive3, while version 5 is compatible with Hive2. See https://github.com/HiveRunner/HiveRunner#hive-version-compatibility